### DISCUSSION TITLE
Welcome to AuthAppForTesla Discussions!

### DISCUSSION THREAD

**Discussion by Kemmey** [TeslaBuds/AuthAppForTesla#12] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Joules53** [comment]
Hi, I'm rather new to this Github / Xcode / Swift area. I've recently upgraded my Mac so that it is now on Monterey (12.0.1), I've downloaded Xcode 13.1, and downloaded AuthAppForTesla from Github. Then I open the AuthAppForTesla.xcworkspace in Xcode and press play. It says Build Succeeded (5 Swift Compiler Warnings) but does not run anything. I can not find any generated app or exe. Please advise what I can do to get AuthAppForTesla to run on my Mac. Thanks.

**Reply by Kemmey** [reply]
Grab it from the Mac app store, if you're not familiar with writing software?

**Reply by Joules53** [reply]
Thanks @Kemmy, I was making the process far too hard for myself.
